src/integration-test/resources/db/mod/reports/PoolAnalysisReportITest_Typical.sql

```diff
@@ -0,0 +1,110 @@
+INSERT INTO juror_mod.pool (owner, pool_no, return_date, total_no_required, no_requested, pool_type, loc_code,
+                            new_request, attend_time)
+VALUES ('400', '415240101', '2024-01-15', 20, 20, 'CRO', '415', 'N', '2024-01-15 00:09:30.000'),
+       ('415', '415240102', '2024-01-15', 20, 20, 'CRO', '415', 'N', '2024-01-15 00:09:30.000'),
+       ('415', '767240101', '2024-01-16', 20, 20, 'CRO', '767', 'N', '2024-01-16 00:09:30.000');
```

Comment:
Should have one pool that is out of selection range

Reply:
added another pool with data